docs/source/notebooks/finch-usage.ipynb

```diff
@@ -110,7 +110,7 @@
      "output_type": "stream",
      "text": [
       "Process status:  ProcessSucceeded\n",
-      "Link to process output:  http://localhost:5000/outputs/8a66a520-5b65-11eb-9d7f-a44cc85b43bf/frost-days_SRES-A2-experiment_20460101-20650101.nc\n"
+      "Link to process output:  https://pavics.ouranos.ca/wpsoutputs/a7195d82-5d01-11eb-a5f7-a44cc85b43bf/frost-days_SRES-A2-experiment_20460101-20650101.nc\n"
```

Comment:
thanks for this.  You used `make refresh-notebooks`?

Reply:
Ehh.. good question, probably not, seeing it uses pavics.ouranos.ca.

Reply:
Oh I thought the switch from `localhost` to `pavics` was because you used `make refresh-notebooks` which would have done the replacement automatically.  So you actually run it against `pavics` manually?   Anyways, I was just curious, everything is fine.